### PR TITLE
SLIM-1010 Fix Handling Of FunctionalExceptions For OperationHolders

### PR DESCRIPTION
**Pull request by riannefijn**

### CODE REVIEW

**Comment by riannefijn** [comment]
retest this please

**Comment by riannefijn** [comment]
retest this please

File: cucumber-tests-platform-smartmetering/src/test/java/com/alliander/osgp/cucumber/platform/smartmetering/glue/steps/ws/smartmetering/smartmeteringinstallation/DeCoupleDeviceSteps.java [hunk 1]
```diff
@@ -104,4 +106,23 @@ public void theDeCoupleResponseIsAndContains(final String status, final List<Str
         assertTrue("Description should contain all of " + resultList,
                 this.checkDescription(response.getDescription(), resultList));
     }
+
+    @Then("^no decouple device response and a FunctionalException are received$")
+    public void noDecoupleDeviceResponseAndAFunctionalExceptionAreReceived(final Map<String, String> settings)
```
Comment (robindenadel) [inline cucumber-tests-platform-smartmetering/src/test/java/com/alliander/osgp/cucumber/platform/smartmetering/glue/steps/ws/smartmetering/smartmeteringinstallation/DeCoupleDeviceSteps.java]: make the method more generic by introducing a parameter for the kind of response

File: cucumber-tests-platform-smartmetering/src/test/java/com/alliander/osgp/cucumber/platform/smartmetering/glue/steps/ws/smartmetering/smartmeteringinstallation/CoupleDeviceSteps.java [hunk 2]
```diff
@@ -103,4 +105,22 @@ public void theCoupleResponseIsAndContains(final String status, final List<Strin
         assertTrue("Description should contain all of " + resultList,
                 this.checkDescription(response.getDescription(), resultList));
     }
+
+    @Then("^no couple mbus device response and a FunctionalException are received$")
```
Comment (robindenadel) [inline cucumber-tests-platform-smartmetering/src/test/java/com/alliander/osgp/cucumber/platform/smartmetering/glue/steps/ws/smartmetering/smartmeteringinstallation/CoupleDeviceSteps.java]: make the method more generic by introducing a parameter for the kind of response

File: cucumber-tests-platform-smartmetering/src/test/java/com/alliander/osgp/cucumber/platform/smartmetering/glue/steps/ws/smartmetering/smartmeteringinstallation/AddDeviceSteps.java [hunk 3]
```diff
@@ -88,6 +91,28 @@ public void theGetAddDeviceResponseShouldBeReturned(final Map<String, String> re
         assertEquals("Result", expectedResult, response.getResult().name());
     }
 
+    @Then("^no add device response and a TechnicalException are received$")
```
Comment (robindenadel) [inline cucumber-tests-platform-smartmetering/src/test/java/com/alliander/osgp/cucumber/platform/smartmetering/glue/steps/ws/smartmetering/smartmeteringinstallation/AddDeviceSteps.java]: make the method more generic by introducing a parameter for the kind of response

**Pull request merged**